pkg/services/live/live.go

```diff
@@ -868,6 +870,73 @@ func (g *GrafanaLive) HandleChannelRulesListHTTP(c *models.ReqContext) response.
 	})
 }
 
+type ConvertDryRunRequest struct {
+	ChannelRules []pipeline.ChannelRule `json:"channelRules"`
+	Channel      string                 `json:"channel"`
+	Data         json.RawMessage        `json:"data"`
```

Comment:
I think this should just be string -- we want to send influx and Prometheus text format also

Reply:
Done, updated PR description